client/src/components/todo/TodoElement.vue

```diff
@@ -184,14 +184,23 @@ const newTitle = ref<HTMLElement | null>(null);
 const newDescriptionMobile = ref<HTMLElement | null>(null);
 const newDescriptionDesktop = ref<HTMLElement | null>(null);
 
+function formatDate(date: string | Date) {
+  const dateObj = new Date(date);
+  const day = dateObj.getDate().toString().padStart(2, '0');
+  const month = (dateObj.getMonth() + 1).toString().padStart(2, '0');
+  const year = dateObj.getFullYear().toString();
+
+  return `${day}.${month}.${year}`;
+}
+
 function saveTodo(todo: TodoType) {
 
     const updatedTitle = newTitle.value?.innerHTML || ''
     let updatedDescription = '';
```

Comment:
you can do const updatedDescription = newDescriptionDesktop.value?.innerHTML || newDescriptionMobile.value?.innerHTML || '' right away